<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>605</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Al Bayan</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Отрезок</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начальная точка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Конечная точка</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Line_algorithm">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>100</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Стандартный метод</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Метод ЦДА</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Брезенхейм с целыми числами</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Брезенхейм с действительными числами</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Брезенхейм с устранением ступенчатости</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Line_color">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>100</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Черный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Красный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Синий</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Фон</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Алгоритм</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>80</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Цвет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LineButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>140</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Al Bayan</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>Отобразить линию</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>170</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>190</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Уголовой шаг</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>190</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Диаметр пучка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>240</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Цвет</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Range_algorithm">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>260</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Стандартный метод</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Метод ЦДА</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Брезенхейм с целыми числами</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Брезенхейм с действительными числами</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Брезенхейм с устранением ступенчатости</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Range_color">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>260</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Черный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Красный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Синий</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Фон</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>240</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Алгоритм</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RangeButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>290</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Al Bayan</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>Отобразить пучок</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>320</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="TimeButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>340</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Al Bayan</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Построить временную гистограмму</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CleanButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>370</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистить экран</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Line_x1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-5000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>5000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Line_y1">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-5000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>5000.989999999999782</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Line_y2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>50</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-5000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>5000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Line_x2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>50</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-5000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>5000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Range_angl">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>210</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Range_diam">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>210</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <double>500.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>430</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Время выполнения алгоритма:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>460</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>400</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Al Bayan</family>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>Line_x1</tabstop>
  <tabstop>Line_y1</tabstop>
  <tabstop>Line_x2</tabstop>
  <tabstop>Line_y2</tabstop>
  <tabstop>Line_algorithm</tabstop>
  <tabstop>Line_color</tabstop>
  <tabstop>LineButton</tabstop>
  <tabstop>Range_angl</tabstop>
  <tabstop>Range_diam</tabstop>
  <tabstop>Range_algorithm</tabstop>
  <tabstop>Range_color</tabstop>
  <tabstop>RangeButton</tabstop>
  <tabstop>TimeButton</tabstop>
  <tabstop>CleanButton</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
